inputs/templates/test/VaPoR/VaPoR.json.tmpl

```diff
@@ -1,15 +0,0 @@
-{
-    "VaPoR.sv_base_mini_docker" : "gatksv/sv-base-mini:b3af2e3",
```

Comment:
Given the expense of VaPoR, it might be nice to have a single-sample test JSON. I have an updated version on my branch, although it doesn't have public data

Reply:
Added a single-sample test json with an "example" sample